.travis.yml

```diff
@@ -5,6 +5,7 @@ python:
   - "3.3"
   - "3.4"
   - "3.5"
+  - "3.6-dev"
```

Comment:
This should be ``3.6``, unless we're explicitly trying to test against the ``HEAD`` of the 3.6 branch (but if so, why are we not doing the same for the other versions still getting maintenance?)